test/apify/request_list.test.ts

```diff
@@ -619,7 +630,7 @@ describe('RequestList', () => {
             reqs.push(request);
         }
 
-        reqs = shuffle(reqs);
+        reqs = shuffle(reqs) as typeof reqs;
```

Comment:
nit: if you add a generic type argument to the function, you won't need to cast anything

e.g. something like this:

```ts
shuffle<T extends unknown[]>(a: T): T { ... }
```